cloud-sql/postgres/sqlalchemy/requirements-test.txt

```diff
@@ -1 +1,3 @@
+-r requirements.txt
```

Comment:
This shouldn't be needed. The test runner should install both requirements.txt and requirements-test.txt.